README.md

```diff
@@ -554,7 +554,12 @@ Ensures jsdoc block exist
 
 Type: `Boolean` or `String`
 
-Values: `true` or `"exceptExports"` (skip `module.exports = function () {};`)
+Values:
+- `true`
+- `{allExcept: ["expressions"]}` (skip expression functions)
+- `{allExcept: ["exports"]}` (skip `module.exports = function () {};`)
+- `{allExcept: ["noParamsAndReturns"]}` (functions without parameters and with empty return statements will be skipped)
+- `allExcept` is an array, so you can combine multiple values
```

Comment:
This better to format like: https://github.com/jscs-dev/node-jscs/blob/master/lib/rules/disallow-comma-before-line-break.js#L8-L10